docs-source/content/faq/boot-identity-not-valid.md

```diff
@@ -17,3 +17,13 @@ the domain directory, which occurs during the "introspection" phase, and when th
 Check that the user name and password credentials stored in the Kubernetes Secret referenced by `weblogicCredentialsSecret` contain the expected values for an account with administrative privilege for the WebLogic domain.
 Then [stop all WebLogic Server instances](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#starting-and-stopping-servers)
 in the domain before restarting so that the operator will repeat its introspection and generate the corrected `boot.properties` files.
+
+If the "Boot identity not valid" error is still not resolved, Managed Server may be failing to communicate with the Administration Server for other reasons. Check carefully for any errors or exceptions logged before the "Boot identity not valid" error in the Managed Server log file and look for indications of communication failure reasons.
```

Comment:
Line 12, 'it means' --> 'it usually means'

Reply:
Rewrite 

`If the "Boot identity not valid" error is still not resolved, Managed Server may be failing to communicate with the Administration Server for other reasons. Check carefully for any errors or exceptions logged before the "Boot identity not valid" error in the Managed Server log file and look for indications of communication failure reasons.` 

as

`If the "Boot identity not valid" error is still not resolved, and the error is logged on a WebLogic Managed Server, then the server may be failing to contact with the Administration Server. Check carefully for any errors or exceptions logged before the "Boot identity not valid" error in the Managed Server log file and look for indications of communication failure.  For example:`